src/setup_payload/java/SetupPayloadParser-JNI.cpp

```diff
@@ -163,6 +167,37 @@ jobject TransformSetupPayload(JNIEnv * env, SetupPayload & payload)
     return setupPayload;
 }
 
+jobject CreateCapabilitiesSet(JNIEnv * env, RendezvousInformationFlags flags)
+{
+    jclass setClass          = env->FindClass("java/util/HashSet");
```

Comment:
nit: "set" as a prefix can easily be miss interpreted here. Rename to "hashSet"? Same with the variables below.

Another example, "setAddMethod" is confusing